cleanlab/filter.py

```diff
@@ -144,24 +143,24 @@ def _prune_by_class(k, args=None):
       The class of interest."""
 
     if args:  # Single processing - params are passed in
-        s, s_counts, prune_count_matrix, psx, multi_label = args
+        labels, label_counts, prune_count_matrix, psx, multi_label = args
     else:  # Multiprocessing - data is shared across sub-processes
-        s, s_counts, prune_count_matrix, psx, multi_label = _get_shared_data()
+        labels, label_counts, prune_count_matrix, psx, multi_label = _get_shared_data()
 
-    if s_counts[k] > MIN_NUM_PER_CLASS:  # No prune if not MIN_NUM_PER_CLASS
-        num_errors = s_counts[k] - prune_count_matrix[k][k]
-        # Get rank of smallest prob of class k for examples with noisy label k
-        s_filter = np.array(
-            [k in lst for lst in s]) if multi_label else s == k
+    if label_counts[k] > MIN_NUM_PER_CLASS:  # No prune if not MIN_NUM_PER_CLASS
```

Comment:
If we're using a > instead of >=, then the actual minimum number per class is 2, right?

Perhaps change MIN_NUM_PER_CLASS to 2, and use >= here for clarity.

Reply:
Relevant for L181-183 as well.